azure-pipelines-official.yml

```diff
@@ -45,6 +45,13 @@ stages:
   displayName: Build and Test
 
   jobs:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
This check isn't strictly necessary, because this YML does not run for PRs.

Reply:
Ty for letting me know, will remove.